treeherder/etl/runnable_jobs.py

```diff
@@ -218,7 +222,14 @@ def list_runnable_jobs(project, decision_task_id=None):
 def _query_latest_gecko_decision_task_id(project):
     url = TASKCLUSTER_INDEX_URL % project
     logger.info('Fetching {}'.format(url))
-    latest_task = fetch_json(url)
-    task_id = latest_task['taskId']
-    logger.info('For {} we found the task id: {}'.format(project, task_id))
+    try:
+        latest_task = fetch_json(url)
+        task_id = latest_task['taskId']
+        logger.info('For {} we found the task id: {}'.format(project, task_id))
+    except requests.exceptions.HTTPError as e:
+        # Specifically handle 404 errors, as it means there's no decision task on this push
+        if e.response.status_code == 404:
+            task_id = ''
```

Comment:
++ to this.

I'd also like a log message at the info level, saying that we didn't find the decision task.